src/containers/landing/index.tsx

```diff
@@ -29,15 +45,21 @@ const Landing: React.FC = () => {
               content="The first page someone sees if they are not logged in, contains a read only map of Boston neighborhoods and some information about the tree counts of Speak for the Trees"
             />
           </Helmet>
-          <MobileMapPage>
-            <MobileLandingBar>
-              <LandingTreeStats
-                moneySaved={statMoneySaved}
-                rainWater={statRainWater}
-                carbonEmissions={statCarbonEmissions}
-              />
-            </MobileLandingBar>
-          </MobileMapPage>
+          {asyncRequestIsComplete(blocks) &&
```

Comment:
🏕️ What does loading look like?

Reply:
The navbar is visible but the rest of the page is blank. We may want to add in loading screens to each of the pages with api requests at some point.